superdesk/vocabularies/vocabularies.py

```diff
@@ -73,7 +73,6 @@ class VocabulariesResource(Resource):
         },
         'service': {
             'type': 'dict',
-            'default': {'all': True},
```

Comment:
Duplicates genre and other dropdowns that had default vocabulary. If the service is present means that it is a custom vocabulary.